content/04.model-complexity.regularization.md

```diff
@@ -0,0 +1,55 @@
+## Regularization {.page_break_before}
+
+Machine learning algorithms are optimized to find patterns among data points. 
+Given a large amount of data, this optimization works in favor of the algorithm trying to learn complex patterns in the data. 
+However, when data is sparse, the same optimization leads to the model learning technical variances rather than actual patterns in the data. 
+This results in low bias but high variance in the prediction results i.e. overfitted models fit the training data very closely (low bias), but fail to predict new testing data with high accuracy (high variance) leading to low prediction accuracy of the model in view of new data. 
+Such overfitting can lead to misinterpretation of technical variances as true patterns in the sparse data and thus needs to be minimized. 
+Minimization of overfitting can be accomplished by crossvalidation and regularization methodologies. 
+While crossvalidation aims to reduce the variance in prediction, regularization adds a small amount of bias to the initial model to minimize its dependance and sensitivity to training data. 
+Regularization makes models less sensitive to variance in sample data by adding a penalty (lambda) to the sum of squared residuals (or sum of maximum likelihoods). 
```

Comment:
This couple of sentences might be a bit hard for a novice-to-ML reader to follow without a figure, or at least citation describing this in more detail. I don't think it's the role of this perspective to teach these concepts outright, but perhaps this could be written in a way that it could be accessible to a less expert reader.

Reply:
reframed and simplified this part in the new commit